scripts/ci-e2e.sh

```diff
@@ -53,7 +53,8 @@ clone_repos
 REPO_ROOT=$(realpath "$REPO_ROOT") 
 # Copy the current CAPM3 repo to the Go source directory
 rm -rf "${M3PATH}/cluster-api-provider-metal3" # To avoid 'permission denied' error when overriding .git/
-cp -R "${REPO_ROOT}" "${M3PATH}/cluster-api-provider-metal3/" 
+# TODO (Mohammed) Remove the next line since it overrides the cloned v1a5 
```

Comment:
I kept this here since I am still not sure why we needed this at the first place I will remove this once squash the commit but I think the comments just bring attention to this changes and make it easy to review

Reply:
cc @namnx228